NativeApp/src/Android/AndroidAppBase.cpp

```diff
@@ -244,10 +359,10 @@ void AndroidAppBase::SetState(android_app* state, const char* native_activity_cl
 
 bool AndroidAppBase::IsReady()
 {
-    if (has_focus_)
-        return true;
-
-    return false;
+    std::lock_guard<std::mutex> give_me_a_name(mutex);
+    return lifecycleFlags_ & APP_STATUS_ACTIVE &&
```

Comment:
This does not look pretty.
`(app_status_ & APP_STATUS_FLAG_ACTIVE) != APP_STATUS_FLAG_NONE`
Would be clearer